include/oneapi/tbb/detail/_assert.h

```diff
@@ -32,6 +33,9 @@ namespace r1 {
   Otherwise call the assertion handler. */
```

Comment:
The comment above looks outdated. Please take a look. Unfortunately, I cannot comment on the untouched lines of the patch due to lack of such feature in GitHub.

Specifically, the line _If assertion handler is null, print message for assertion failure and abort. Otherwise call the assertion handler._
I think it needs to be revised to something like: _If assertion handler is not set, the default one is used, which prints the message for assertion failure and aborts. The setting of an assertion handler is available as part of oneTBB specification extension._